.github/workflows/build.yml

```diff
@@ -20,18 +20,12 @@ jobs:
       fail-fast: false
       matrix:
         major_version: [37, 38]
-        driver_version: [520, 525, 530, 470]
+        driver_version: [470, 530, 535]
```

Comment:
Removes the v520 and v525 drivers since they are being excluded for both Fedora versions